cmd/up.go

```diff
@@ -1042,7 +1042,7 @@ func setupLoftPlatformAccess(context, provider, user string, client client2.Base
 	cmd.Stderr = &errb
 	err = cmd.Run()
 	if err != nil {
-		log.Errorf("failure in setting up Loft Platform access: %s", errb.String())
+		log.Debugf("failed to set up platform access in workspace: %s", errb.String())
```

Comment:
I think this should be formatted as `%w, errb`

Reply:
Since we don't return there's no need to wrap into `%w` imo